content/Accesibility in React: tools and practices.md

```diff
@@ -0,0 +1,152 @@
+
+This tool in his new version, apart of analyzing statically the code of your application, also _analyzes in run-time_, that means, if for example, the gradient of a screen is not sufficient for people with visual disabilities, the linter will throw a warning when you are running your application.
+
+For integrate this tool, first, you need to add as a dev dependency in your project
+
+```console
+yarn add react-axe --dev
+```
+
+then, you have to initialize it
+
+https://gist.github.com/nsantos16/ff2d9eb40be4541ba9af577c0dd70e6f
+
+And done! You have an accessibility linter integrated into your project.
+
+### Buttons!
+
+Buttons are a complex part of the accessibility, the best way to improve buttons for accessibility is your design team involve practices in the development process.
+
+Here in [Rootstrap](<[https://www.rootstrap.com/](https://www.rootstrap.com/)>) we have guidelines about accesibility, you can check [here](<[https://www.figma.com/file/BsH7BBDNKPBQnaKdunKtJg/RS-UI-Global-Template](https://www.figma.com/file/BsH7BBDNKPBQnaKdunKtJg/RS-UI-Global-Template)>)!
+
+#### States
+
+![Button states](https://i.ibb.co/N7Cxy7K/Captura-de-Pantalla-2020-07-24-a-la-s-17-59-19.png)
+
+The states of the buttons in React are very important, three states are essential for correct behavior on accessibility, there are:
+
+- **Focus**: When our user puts the target on a button, is necessary to give the feedback through the color of the border, this allows users with visual disabilities to have a reference.
+
+- **Textual label**: This with focus events, is very important for interpreter programs, overall for visual and neurological disabilities. Also, is very useful for people who don't understand what means and do the button.
+
+- **Hover**: Change the shape of the button when the hover event happens to indicate in a quick way how button works.
+
+#### Touch target
+
+![enter image description here](https://i.ibb.co/fQB6NhR/Captura-de-Pantalla-2020-07-24-a-la-s-18-09-01.png)
+
+People with low vision or with any kind of motor impairment may struggle to aim for the target, so every actionable element should have a touch target of 48 x 48 px. as a minimum.
+
+#### Naming icons
+
+![enter image description here](https://i.ibb.co/fH7Gtmy/Captura-de-Pantalla-2020-07-24-a-la-s-18-11-13.png)
+
+People with vision impairments listening to screen readers will need an actionable copy to understand the interaction that every icon represents.
+
+There are a few do/don't points we need to follow:
+
+Do:
+
+- Naming an icon based on the action it will perform.
+
+- Using infinitive verbs to indicate an action.
+
+- Clarify the object of the action (Create Account, See More Articles, Download Guide) or details that are critical, such as the price of a purchase.
```

Comment:
```suggestion
- Clarifying the object of the action (Create Account, See More Articles, Download Guide) or details that are critical, such as the price of a purchase.
```

Reply:
I think `Do name, Do use, Do clarify` reads better than `Do Naming, Do using, etc`
Same for `Don'ts`